app/adapters/alma_adapter/alma_item.rb

```diff
@@ -151,7 +151,7 @@ def recap_use_restriction
     end
 
     def group_designation
-      return unless item.library == "recap"
+      return unless recap_item?
       case item.location
       when 'pa', 'gp', 'qk', 'pf'
```

Comment:
Are these still recap locations? If so they should be added to the "all_recap_groups" method.

Reply:
@eliotjordan they are look https://bibdata-alma-staging.princeton.edu/locations/holding_locations but I think we should adjust the location name to be recap$pa, recap$gp etc

Reply:
https://api-na.hosted.exlibrisgroup.com/almaws/v1/conf/libraries/recap/locations?apikey=apikey&format=json

Reply:
You mean all locations? 

```
 d['pj', 'pk', 'pl', 'pm', 'pn', 'pt']  
["pb", "ph", "ps", "pw", "pz", "xc", "xg", "xm", "xn", "xp", "xr", "xw", "xx"]  
['jq', 'pe', 'pg', 'ph', 'pq', 'qb', 'ql', 'qv', 'qx'] 
```

These should all be prefixed by "recap$"?

Reply:
Some will be prefaced by recap$, those are the ones that will still remain a part of the "Recap" library in Alma, but those are moving to be filed under a branch library but flagged for "remote_storage"  an example of this would be "pk" which will come through as "mendel$pk" under the new remote_storage set-up.

Reply:
For now I think the way to handle this is to prefix all the locations in these arrays with "recap$". I'll give you a list of the new "remote_storage" at the branch library locations.

Reply:
Are you sure? I thought the point of doing it this way was so that we would catch things that don't have a recap location because of the switch to remote storage.